source/_patterns/components/secondary-navigation/secondary-nav.hbs

```diff
@@ -1,6 +1,6 @@
   <ul class="pf-secondary-nav" role="tablist">
     <li class="pf-secondary-nav__item">
-      <a href="#" role="tab" class="pf-secondary-nav__link" aria-selected="true">Secondary Tab One</a>
+      <a href="#" role="tab" class="pf-secondary-nav__link is-active" aria-selected="true">Secondary Tab One</a>
```

Comment:
`active` is more general. a card can be active but not selected. It's easier to reuse I think